### PR TITLE
Group Woes

### PR DESCRIPTION
**Pull request by bbesmanoff**
This seems to be working enough to be exciting.  Groups list shows properly in the list (sometimes with a delay) and the overlay updates properly.

### CODE REVIEW

**Comment by thenaterhood** [comment]
Is this up to date with your previous pull to remove the group store? There's a couple spots that it seems like it may not be.

**Comment by bbesmanoff** [comment]
It is rebased against 32cbe5ef94745319327733dc6cbcc33d2d8ec904, so it should be.  What are you seeing?

File: app/controller/Groups.js [hunk 1]
```diff
@@ -82,6 +82,7 @@ Ext.define('DevCycleMobile.controller.Groups', {
 		this.groupStore.filter('groupCode', code);
```
Comment (thenaterhood) [inline app/controller/Groups.js]: I was noting this and the several lines below it that reference a groupStore.

Comment (bbesmanoff) [inline app/controller/Groups.js]: Ah - my previous pull removed the rider store.  The group store is still in use for holding/displaying the groups

Comment (thenaterhood) [inline app/controller/Groups.js]: Oh gotcha gotcha, I got mixed up. So groupStore and groupRiderStore are still kosher. I'll test this real quick and give you a european post office.

**Comment by thenaterhood** [comment]
There's room for improvement on the Groups page refreshing as I've mentioned, but this is all good by me. :european_post_office:

**Comment by CShapleigh** [comment]
I will test this as soon as I get home, then I'll confirm or deny...unless someone else does it before me

**Comment by thenaterhood** [comment]
If you need a cable I can provide that now.

**Comment by CShapleigh** [comment]
CONFIRMED. BEAUTIFUL. FANTASTIC.

**Pull request merged**